### PR TITLE
Makes Pets faster, cat improvements.

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
Makes station pets the same speed as people running.

lets cats sit down, using *sit or an in game verb.
when resting they can rest left or right now, same with sitting.

## Why It's Good For The Game
AI cats can sit, player cats now can, 
Being able to rest multiple ways makes sense.

being faster is nice so it doesnt take years to get anywhere as a pet. they cannot drag bags/people so its not harmful.

## Images of changes
dreamseeker_Cbhzf0eTBb
image

## Testing
images of changes p-much

## Changelog
:cl:
tweak: station pets (except Sergeant Araneus) are now as fast as humans running
add: player cats can sit now with a keybind, verb, or emoting *sit
add: sprites for cats sitting and resting to the left.
/:cl:

### CODE REVIEW

**Comment by Denthamos** [comment]
The laying down/sitting sprite is a bit off and has it in the turf below it.  Should be brought up a few pixels to be kept on the same turf.

File: code/modules/mob/living/simple_animal/friendly/friendly_emote.dm [hunk 1]
```diff
@@ -91,3 +91,12 @@
 	key = "purr"
 	key_third_person = "purrs"
 	message = "purrs."
+
+/datum/emote/living/sit/cat
```
Comment (lewcc) [inline code/modules/mob/living/simple_animal/friendly/friendly_emote.dm:95]: You can just make this `/datum/emote/living/simple_animal/pet/cat/sit`, and you won't need to update the `mob_type_allowed_typecache`. See above.

File: code/modules/mob/living/simple_animal/friendly/friendly_emote.dm [hunk 2]
```diff
@@ -91,3 +91,12 @@
...
+	message = null
```
Comment (lewcc) [inline code/modules/mob/living/simple_animal/friendly/friendly_emote.dm:96]: Also, you should probably set a key (and key_third_person) for good measure, so people can emote *sit or *sits.

**Review by lewcc — COMMENTED** [review]
Some small emote things.

**Review by S34NW — COMMENTED** [review]
Seems fine apart from Lewcc's changes

File: code/modules/mob/living/simple_animal/friendly/friendly_emote.dm (hunk 2 above)
Comment (lewcc) [reply code/modules/mob/living/simple_animal/friendly/friendly_emote.dm:96]: Been made aware that there already is a `/datum/emote/living/sit`, and so this inheritance pulls from that. Oops.

**Review by lewcc — APPROVED** [review]
Been made aware that there already is a /datum/emote/living/sit, and so this inheritance pulls from that. Oops.
With that in mind, looks good to me.

**Review by farie82 — APPROVED** [review]

**Pull request merged**